📜Create tutorials (#4)
KinsonDigital/VelaptorDocs: docs/tutorials/guides/Coming Soon/fonts.mdx

---
sidebar_position: 7
---

# Fonts

Documentation coming soon!!
:::note

> This feature exists and can probably be figured out by reading the API documentation.

Reviewer: Refactor to below:
```suggestion
This feature currently exists and is explained in the API documentation.
```
